packages/openapi-ts/src/ir/context.ts

```diff
@@ -88,7 +88,7 @@ export class IRContext<Spec extends Record<string, any> = any> {
    */
   public createFile(file: ContextFile): GeneratedFile {
     // TODO: parser - handle attempt to create duplicate
-    const outputParts = file.path.split('/');
+    const outputParts = file.path.split(path.sep);
```

Comment:
Are you on Windows? Was this causing any trouble?

Reply:
I'd be wary of this change. Tests weren't failing before this pull request so I assume this is causing them to fail. But the thing I am more concerned about is that plugins always define their path using slash `/` so if you change this based on the system, it will break a lot of stuff.